src/components/pages/exportcontent/contentcatalogue.js

```diff
@@ -0,0 +1,21 @@
+import React, { useState, useEffect } from 'react';
+import { useLocation } from '@reach/router';
+import { buildQueryString, parseQueryString } from '~utils/query-string';
+
+const COMPOSITESEARCHURL = `https://us-east-1.aws.webhooks.mongodb-realm.com/api/client/v2.0/app/devhub-search-service-eybnu/service/devhub_export/incoming_webhook/compositesearch`;
+
+const ContentCatalogue = () => {
+    const { search } = useLocation();
+    const { ...params } = parseQueryString(search);
+    const queryParams = buildQueryString(params);
+    const [searchResult, setSearchResult] = useState(null);
+    useEffect(() => {
+        const url = `${COMPOSITESEARCHURL}${queryParams}`;
+        fetch(url)
+            .then(response => response.json())
+            .then(data => setSearchResult(data));
+    }, []);
```

Comment:
My one other thought is if we want to block this page from rendering until this piece is finished. My thinking is that it would be easier to scrape since on load the data will be there instead of a re-render. What do you think?

For example, try to `view source code` for this page versus the home page?